Dockerfile.tmpl

```diff
@@ -268,9 +257,15 @@ RUN pip install scipy \
     apt-get install -y graphviz && pip install graphviz && \
     # Pandoc is a dependency of deap
     apt-get install -y pandoc && \
-    pip install git+https://github.com/scikit-learn-contrib/py-earth.git@issue191 \
-        essentia && \
-    apt-get install -y git-lfs && \
+    pip install essentia
+
+# http://go/roole/git_clone_github.md
+#RUN git config --global --add url.https://github.com/.insteadOf git://github.com/ && \
+#     git clone git://github.com/scikit-learn-contrib/py-earth.git && \
+#        cd py-earth && \
+#        python setup.py install
```

Comment:
I'll take a pass through and cleanup commented out things, leaving for now so I don't have to rebuild/retest since that finally passed.